network.go

```diff
@@ -575,7 +576,21 @@ func (s *sandbox) updateRoute(netHandle *netlink.Handle, route *types.Route, add
 // DNS //
 /////////
 
-func setupDNS(dns []string) error {
+func setupDNS(dns []string) (err error) {
+	var file *os.File
+	GuestDNSFile := "/etc/resolv.conf"
```

Comment:
this variable should be declared outside this function and not exported (the first character shouldn't be a capital letter" `guestDNSFile`), that way you can write a unit test